docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -157,11 +188,41 @@ Here are the steps to configure a new Google Service Account Key in EAS for send
 </Step>
 
 <Step label="4">
+  Configure the `google-services.json` file in your app. (Note: you may skip this step if `google-services.json` has already been set up.)
+
+  First, download the `google-services.json` file from Firebase Cloud Console and place it somewhere in your app directory (the root is fine). 
+
+  <ImageSpotlight
+    alt="Download google-services.json from Firebase Cloud Console"
+    src="/static/images/creating-google-service-account/fcm-v1/existing-service-account/04-setup-google-services-json.png"
+    style={{ maxWidth: 800 }}
+  />
+
+  Then update your app.json with the key `expo.android.googleServicesFile` and value `path/to/google-services.json`.
+
+  <CodeBlocksTable>
+    ```json
+	// app.json
+	{
+	  "expo": {
+		...
+		"android": {
+		  ...
+		  "googleServicesFile": "./path/to/google-services.json"
+		},
+	  }
+	}
+    ```
+  </CodeBlocksTable>
```

Comment:
````suggestion
    ```json app.json
    {
      "expo": {
    	/* @hide ...*/ /* @end */
    	"android": {
    	  /* @hide ...*/ /* @end */
    	  "googleServicesFile": "./path/to/google-services.json"
    	},
      }
    }
    ```
````